src/modules/cmdpal/exts/PokedexExtension/Properties/launchSettings.json

```
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "MastodonExtension (Package)": {
```

Op this needs to say 
```suggestion
    "PokedexExtension (Package)": {
```

(and below)

Otherwise folks get real confused